src/test/regress/sql/multi_explain.sql

```diff
@@ -108,6 +108,10 @@ SELECT true AS valid FROM explain_xml($$
 SELECT true AS valid FROM explain_json($$
 	SELECT l_quantity FROM lineitem WHERE l_orderkey = 5$$);
 
+-- ensure EXPLAIN EXECUTE doesn't crash
+PREPARE router_executor_query AS SELECT l_quantity FROM lineitem WHERE l_orderkey = 5;
+EXPLAIN EXECUTE router_executor_query;
```

Comment:
If we just add this to the bottom, does that prevent all the other changes?

Maybe also add a real-time executor query.